### PR TITLE
Refactor/penrose-background

### PR DESCRIPTION
**Pull request by queuing4oranges**
- adding new component to utils that will generate the Penrose background and resize the background when the window size changes
- adding the `generatePenrose`to all relevant screens

"can't login" screen and resizing

https://github.com/TeskaLabs/seacat-auth-webui/assets/31582696/b156e4c0-005d-4c2a-af8a-5b83f08456b9

all screens

https://github.com/TeskaLabs/seacat-auth-webui/assets/31582696/cc7ce2d9-94e4-4aca-ab09-ded62330647b

different browser /testing resizing

https://github.com/TeskaLabs/seacat-auth-webui/assets/31582696/2d766278-0179-417b-8504-e4e039945923

### CODE REVIEW

File: src/modules/auth/containers/LoginScreen.js [hunk 1]
```diff
@@ -59,31 +53,6 @@ function LoginScreen(props) {
 		}
 	}
 
-	function useWindowSize() {
```
Comment (queuing4oranges) [inline src/modules/auth/containers/LoginScreen.js:62]: the `useWindowSize` is only being used to measure and for re-creating background when screen size changes (same was in RegisterScreen) 

it's now part of the `generatePenrose` function

File: src/modules/auth/utils/generatePenrose.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+			if (bgScript) bgScript.remove();
+				const removeBackground = generateBackground();
+			return () => {
+				removeBackground();
+		};
+
+	}, [size])
+
+	return null; 
+}
```
Comment (Pe5h4) [inline src/modules/auth/utils/generatePenrose.js]: @queuing4oranges No new line at the end of the file

**Review by Pe5h4 — CHANGES_REQUESTED** [review]
Alright, this looks very good. Please test that also on invitation (but this should look the same I suppose)

**Review by ateska — APPROVED** [review]

**Comment by queuing4oranges** [comment]
- tested on invitation
- added also to password reset

penrose-invitation-diffuser

penrose-invitation

penrose-reset-pw

penrose-reset-pw2

**Review by Pe5h4 — APPROVED** [review]

**Pull request merged**